plugins/trogonstack-eventmodeling/skills/eventmodeling-translating-external-events/SKILL.md

```
@@ -0,0 +1,409 @@
+**Source System**: [System name]
+
+**Technical Representation**:
+```json
+{
+  "field1": "value",
+  "field2": "value"
+}
+```
+
+**Domain Translation**:
+| External Field | Our Field | Mapping | Notes |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)